doc/research/05_acting.md

```diff
@@ -0,0 +1,199 @@
+# Basic research acting
+
+**Summary:** On this page you can find the results of the basic research on acting.
+
+---
+
+### Authors
+
+Gabriel Schwald, Julian Graf
+
+### Date
+
+14.11.2022
+
+---
+[[TOC]]
+
+## Objective
+
+The job of this domain is to translate a preplanned trajectory into actual steering controls for the vehicle.
+
+- safety:
+  - never exceeding vehicle limits
+  - never exceeding speed limits
+  - never leaf path
+- driving comfort?
+
+## Solutions from old PAF projects
+
+### [Paf 20/1](https://github.com/ll7/psaf2/wiki/Path-Tracking-Algorithmen)
+
+- modified [carla_ackermann_control](https://carla.readthedocs.io/projects/ros-bridge/en/latest/carla_ackermann_control/)
```

Comment:
auf welche Art modifiziert?

Reply:
Der PID-Regler wurde getuned

Reply:
Die Ackermann-Node wurde verändert, dass sie twist-msgs akzeptiert

Reply:
Fixed in a233871ea28ed887a157136892b2155cea45d647